controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -2656,9 +2656,6 @@ private StreamSegmentRecord newSegmentRecord(int epoch, int segmentNumber, long
      */
     abstract CompletableFuture<List<VersionedTransactionData>> getOrderedCommittingTxnInLowestEpoch(
             int limit, OperationContext context);
-    
-    @VisibleForTesting
```

Comment:
why is this method removed?

Reply:
I had to make this method public to call it from CommitRequestHandler hence remove from here and add to Stream class, but I can reverse all these changes now.

Reply:
fixed